### PR TITLE
Fuzzy search: Add position feature to the ngram analyzer

### PR DESCRIPTION
**Pull request by semberal**

### CODE REVIEW

**Comment by joerg84** [comment]
@semberal Thanks, could you sign the CLA (you know German Law ;-)) https://www.arangodb.com/community/
Danke!

**Review by cw00dw0rd — APPROVED** [review]

**Comment by cw00dw0rd** [comment]
LGTM, thank you! Once CLA let's merge.

**Comment by ansoboleva** [comment]
Fixed as part of https://github.com/arangodb/docs/pull/854.

**Pull request closed without merging**
